test/e2e/export/simple_test.go

```diff
@@ -0,0 +1,60 @@
+package e2e_export_test
+
+import (
+	"strings"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/werf/werf/v2/test/pkg/werf"
+)
+
+var _ = Describe("Simple export", Label("e2e", "export", "simple"), func() {
+	It("should succeed and export images",
```

Comment:
You need to write two separate tests instead of one:
- It should succeed and export image;
- It should succeed and export image (multi-platform).

Two-step fixtures (state0, state1) aren’t necessary in this test; instead, a single directory setup like [this example](https://github.com/werf/werf/blob/aa63823a83c29cf3439db413703345210883bf26/test/e2e/build/_fixtures/cmd_entrypoint) will be sufficient. For the multi-platform test, simply add the `--platform=linux/amd64,linux/arm64` option to achieve the required behavior.